src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -250,12 +255,37 @@ export const thorchainSaversStakingOpportunitiesUserDataResolver = async ({
       const rewardsAmountsCryptoBaseUnit: [string] = [
         stakedAmountCryptoBaseUnitIncludeRewards.minus(stakedAmountCryptoBaseUnit).toFixed(),
       ]
+      let maturity: number | undefined = undefined
```

Comment:
```suggestion
```

No need to define a let and reassign

<img width="1017" alt="image" src="https://github.com/user-attachments/assets/092e2603-9d18-48ba-bb09-dda23e4ad2b9">

Reply:
Fixed in [60e41ac](https://github.com/shapeshift/web/pull/7472/commits/60e41aceb721592767aac449c8b1671168120425)